### ISSUE TITLE
Default Checked State for Checkboxes

### ISSUE THREAD

**Issue by cherrykoda** [verbb/formie#707]
**Description**

Selecting the "default" option on checkboxes is not causing checked="true" on checkbox markup.

**Steps to reproduce**
1. Create a form 
2. Create a checkbox field, select the "default" option
3. render the form to the frontend

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Default
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.5
- Craft version: Craft Pro 3.7.28
- Multi-site: No

**Additional context**

Formie default styling is turned off.

![image](https://user-images.githubusercontent.com/3789834/148619774-f38aa41e-ce75-4abf-89d0-bf517a5da2ac.png)

![image](https://user-images.githubusercontent.com/3789834/148619799-3d0fa479-01ae-4ffe-8064-cc544957bfb0.png)

![image](https://user-images.githubusercontent.com/3789834/148619853-a9bec04e-317c-4ab5-a908-557055e4621e.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by cherrykoda** [comment]
Looks great, thanks for the quick fix @engram-design!

![image](https://user-images.githubusercontent.com/3789834/148895437-ecb591a0-2623-4f60-a1dd-dea78684e440.png)

**Comment by engram-design** [comment]
Fixed in [1.5.6](https://github.com/verbb/formie/releases/tag/1.5.6)
